modules/topicsFpdModule.js

```diff
@@ -15,10 +15,13 @@ let LOAD_TOPICS_INITIALISE = false;
 const HAS_DEVICE_ACCESS = hasDeviceAccess();
 
 const bidderIframeList = {
-  maxTopicCaller: 1,
+  maxTopicCaller: 2,
```

Comment:
@jlquaccia hopefully this change of default is fine with you

Reply:
@piwanczak know this has already been merged, but yea seems alright to me.  Thanks for the heads up!  I took some time off last week, so was just getting around to seeing this PR today.